src/test/java/org/springframework/data/r2dbc/repository/query/StringBasedR2dbcQueryUnitTests.java

```diff
@@ -237,6 +238,23 @@ public void skipsNonBindableParameters() {
 		verifyNoMoreInteractions(bindSpec);
 	}
 
+	@Test // gh-373
+	public void bindsMultipleSpelParametersCorrectly() {
```

Comment:
This test could be implemented directly on `ExpressionQuery` in an `ExpressionQueryUnitTests` where it could be significantly simpler because the transformed query and the `ParameterBinding` values are directly available.

The same seems true for many other tests in this class as well.

Reply:
Done.